### DISCUSSION TITLE
sudo command to reconnect wifi client

### DISCUSSION THREAD

**Discussion by PapaSchIumpf** [RaspAP/raspap-webgui#1605] [Q&A] [answered]
hi everyone,

i have raspap running on a pi 5, i use it as AP for iot devices and have it also connectet to another wifi network with an usb wifi adapter(wlan1)

the problem is that after some time the connection to the wifi client is lost.

in that case i need to connect to the AP again, go to settings over ui and reconnect again to wifi client.

the credentials are known, i only need to press connect, isnt there a sudo command for that?

i could program that a command like this is triggered in an intervall to make sure the connection is stable

thanks for help

**Comment by billz** [comment]
There is indeed, it's done with `wpa_cli`. The command `sudo wpa_cli -i [interface] select_network [network ID]` can be used to (re)connect to an existing network. To determine the network ID value, you can run `wpa_cli` interactively.


```
$ sudo wpa_cli

Selected interface 'wlan0'

Interactive mode

> list_networks
network id / ssid / bssid / flags
0	some_network	any	
1	another_network	any
```

**Reply by PapaSchIumpf** [reply]
i tried with this command, but i get fail
sudo wpa_cli -i [wlan1] select_network [0]
Failed to connect to non-global ctrl_ifname: [wlan1]  error: No such file or directory

i have set a static ip for wlan1 and in /etc/dnsmasq.d this is saved under 090_wlan1
# RaspAP wlan1 configuration
interface=wlan1
dhcp-range=,,0.0.0.0,

**Reply by billz** [reply, marked as answer]
Brackets `[]` are a standard way of indicating placeholders. Use `sudo wpa_cli -i wlan1 select_network 0` instead. See also `man wpa_cli`.

**Reply by PapaSchIumpf** [reply]
thank you, now it works
